drivers/modem/hl78xx/mqtt_offload/Kconfig.mqtt_offload

```diff
@@ -0,0 +1,72 @@
+# MQTT Offload Library for Zephyr
```

Comment:
I just wonder why the MQTT offload API is placed to the driver sources, it sounds like better place for it would be as an external module that the user can then install if necessary.
Zephyr already has MQTT API and I do not think we should introduce another one for MQTT use.